### ISSUE TITLE
Check for a new version of a project when it gets loaded

### ISSUE THREAD

**Issue by wonder-sk** [MerginMaps/mobile#1769]
When loading a project (when the app starts or when the project is selected), it would be nice if Input checked whether the project is at the latest version, and if not, we would show a notification.

Related to #1566

**Comment by PeterPetrik** [comment]
I think this is now the case already?

**Comment by gioman** [comment]
> We don't show notification on current version `2025.8.0`, but users can turn on autosync, which will synchronize the project on opening.

@Withalion there are cases where people for some reason have turned off auto-synch and maybe forgot to turn it on. Warning users that there is a new version of the project on server and asking to update is **very** important. Of course this is about the last project opened, then one that auto-opens when the app starts.

I can't recall (I may be wrong of course) about any warning about a new version of the App being available. This is also very important.

**Comment by gioman** [comment]
> [@Withalion](https://github.com/Withalion) there are cases where people for some reason have turned off auto-synch and maybe forgot to turn it on. Warning users that there is a new version of the project on server and asking to update is **very** important. Of course this is about the last project opened, then one that auto-opens when the app starts.
> 
> I can't recall (I may be wrong of course) about any warning about a new version of the App being available. This is also very important.

@Withalion as far as I have understand this feature is in the pipeline now, right?

**Comment by Withalion** [comment]
Yes @gabriel-bolbotina worked on this issue in #4386. It will be probably released in `2026.4.0`
